tests/backup/backup_test.go

```diff
@@ -922,14 +922,14 @@ var _ = Describe("{ShareBackupWithUsersAndGroups}", func() {
 		Step("Add users to group", func() {
 			log.InfoD("Adding users to groups")
 			var wg sync.WaitGroup
-			for i := 0; i < len(users); i++ {
+			for i, user := range users {
 				groupIndex := i / groupSize
```

Comment:
Isn't it i % groupSize?

Reply:
No. We want the first 10 users to be in groups[0], the next 10 to be in groups[1] and the next 10 to be in groups[2].
Which is why we should not change the `groupIndex` from 0 to 1 to 2 until 10 users are processed.

Reply:
Makes sense.